### ISSUE TITLE
Changing the font on your outlines

### ISSUE THREAD

**Issue by scripting** [scripting/drummerRFC#23]
New feature documented in this [change note](http://scripting.com/drummer/blog/2022/03/04.html#a170046). 

If you have questions or comments post them here.

**Comment by akaKenSmith** [comment]
Running smoothly here on the web version...

**Comment by akaKenSmith** [comment]
No urgency about this, for sure, but if one or two additional font choices were within reach, I'd suggest:

1. The ability to display the blog title at the top of the Old School web page with both lower and upper case letters. (All caps is usually a little less readable in most fonts.)
2. A choice of fonts for displaying the title there.

It's all running smoothly here on the web version, as usual. Merci...

**Comment by scripting** [comment]
Ken, I don’t understand what you’re asking for. Provide a link or an example or screen shot. This is also not related to this feature, I think you’re talking about web rendering. You can also leave out the part about no urgency.

**Comment by akaKenSmith** [comment]
You're right, it's about web rendering. In the image here, I have both lower and upper case letters in the blog's title. And I have a different font as well, just for the title. If I had more skill I would have added the ultra-fine black outline to the letters that already is part of the blog title display in Old School--that's a very nice touch.

![OBx0xwavSNiqoBXsNGZW6Q_thumb_a7d5](https://user-images.githubusercontent.com/4759847/156958523-0ec69780-ea1d-4847-b8d4-3ca1dcdd981c.jpg)

**Comment by scripting** [comment]
Ken I still don’t know what you’re asking for.

**Comment by scripting** [comment]
Ken and I spoke offline and I did finally understand, and found the issue, and will think about how to address it.
